src/Validations.tsx

```diff
@@ -25,3 +24,57 @@ export const loginValidation = (login: string, password: string) => {
   }
   return true;
 };
+
+const emailPattern = /\S+@\S+\.com/;
+const alertMessage = 'Por favor, preencha todos os campos!';
+
+export const newUserValidation = (
+  name: string,
+  email: string,
+  password: string,
+  phone: string,
+  birthDate: string,
+  role: string,
+) => {
+  if (name === '') {
+    alert(alertMessage);
+    return false;
+  }
+
+  if (email === '') {
+    alert(alertMessage);
+    return false;
+  }
```

Comment:
essas verificações de vazio poderiam ser colocadas em um unico if em, também poderia usar o [Logical NOT](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT) para verificar falsy